### PR TITLE
You Can Now Craft The Double Extended Capacity Emergency Oxygen Tank

### PR DESCRIPTION
**Pull request by ghost**
How many of you actually knew this tank even existed

## About The Pull Request
Adds a crafting recipe for the double extended capacity emergency oxygen tank.
You need two extended capacity emergency oxygen tanks, a pipe, and a welding tool to make it.

## Why It's Good For The Game
The tank looks pretty epic, but it's only used in like, two places in the entire game. What if we had an easier way to get it that doesn't involve mugging an ert or going to a single specific ruin in deep space.

## Changelog
:cl:
add: a way to craft the double extended capacity emergency oxygen tank
/:cl:

### CODE REVIEW

**Comment by Ryll-Ryll** [comment]
You should make it burst if there's too much pressure inside while crafting since

1. It adds a bit of work needed to empty, craft, and refill the canisters considering this is basically a straight upgrade to normal emergency tanks
2. Cutting into a pressurized canister with a welding torch using your hands is the poggiest of pog moments

**Comment by RaveRadbury** [comment]
I think this has been done before, closed for powercreep.

**Comment by kriskog** [comment]
These used to be commonplace, but were removed when tank volumes were reduced a while ago.

**Comment by CRITAWAKETS** [comment]
its actually on deltastation but not on the other stations

also the recipe should require an airlock painter, so you can paint the oxygen tanks yellow

**Comment by Paxilmaniac** [comment]
the tanks you craft it with are already yellow, why would you need to paint them

**Comment by kriskog** [comment]
@81Denton You removed these initially right? How do you feel about this one?

**Comment by anconfuzedrock** [comment]
a regular extended emergency tank has about 30 minutes of air.
just carry two.

**Comment by necromanceranne** [comment]
You should see plasmamen tank capacities, that shits the real powercreep.

**Comment by kriskog** [comment]
This tank would be 33% bigger than the plasmaman tanks

**Comment by RaveRadbury** [comment]
Yeah these can last you a whole shift.

**Comment by lolman360** [comment]
ok hear me out: what if it needed an anomaly core

vortex anom because vortices are air related

**Comment by Paxilmaniac** [comment]
that clearly seems too easy to craft, i'll make it take every anomaly core in the game to craft

**Comment by 81Denton** [comment]
@kriskog These tanks are cool, but they're just too big for something you can shove in a suit pocket. The reason they were removed from anything except ERT/deathsquad was because they'd just last you the whole round.
tl;dr I'd prefer it if these weren't available to normal crew.

**Comment by Rohesie** [comment]
Sorry, these tanks look a bit too powerful to be crafted by mere mortals.
If you want to switch this to a weaker version some maintainer approves, do the change and comment on the PR with who approved it, and we'll re-open it.
Or make a new PR.
